CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # bedrock-web-vc-issuer ChangeLog
 
+## 3.0.0 - TBD
+
+### Changed
```

Comment:
It looks like we won't be making a 2.3.0 release after all.  So, please incorporate my comments there into what you've written here.

Please add the typical verbiage that the HTTP client stuff is breaking because of the error API variances.

Aside from that, none of the other changes are breaking?  So for instance, the defaults for `webkms-client` etc around signatures suites won't cause breakage.  So imagine that you'd want to use this new v3.0.0 with an existing issuer deployment without upgrading any KMS or EDV.  If any of these new client libraries are going to use contexts or types that the KMS doesn't understand, then that would be breaking and should be listed as such.

Reply:
Updated it here 71bac83